js/search.js

```diff
@@ -995,6 +995,8 @@
 					var baseTemplate = Tools.getTemplate(template.baseSpecies);
 					for (var j = 0; j < template.types.length; j++) {
 						if (template.battleOnly) continue;
+						// Temporary until all in-battle formes are supported
+						if (template.forme === 'Ultra' || template.forme === 'Pirouette' || template.forme === 'Zen' || template.baseSpecies === 'Castform') continue;
```

Comment:
...?

Where is `template.battleOnly` set?

Reply:
Looks like it’s one line above

Reply:
BattleOnly currently only covers mega evolutions and primals

Reply:
@AustinXII I was asking for where it's set, not where it's read. :|

@KrisXV Sounds like that should be fixed.